feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/store/SessionPagesStore.kt

```diff
@@ -119,6 +119,14 @@ class SessionPagesStore @Inject constructor(
         .map { it.sessionPage }
         .toLiveData(SessionPage.pages[0])
 
+    val reselectedTab: LiveData<Int> = dispatcher
+        .subscribe<Action.SessionPageReselected>()
+        .map { action ->
+            filteredSessions.value
+                ?.filter { session -> session.dayNumber == action.dayNumber }
+                ?.indexOfFirst { session -> session.isOnGoing } ?: -1
+        }.toLiveData(-1)
```

Comment:
If you don't use `toSingleLiveData`, Always scroll to the current time when app become onStart.
What do you think?

Reply:
Sorry... I will fix it!